tests/ecc_test.c

```diff
@@ -117,7 +117,7 @@ static int _ecc_test_shamir(void)
 }
 #endif
 
-static int _ecc_issue108(void)
+static int _ecc_issue108(void) /* https://github.com/libtom/libtomcrypt/issues/108 */
```

Comment:
can you please put this comment above the function

Reply:
done